packages/shared/package.json

```diff
@@ -101,6 +101,7 @@
   "dependencies": {
     "@growthbook/growthbook": "https://gitpkg.now.sh/dailydotdev/growthbook/packages/sdk-js?b8f31f9e80879fe2bcc42b275087b50e1357f1cb",
     "@growthbook/growthbook-react": "^0.17.0",
+    "@paddle/paddle-js": "^1.3.2",
```

Comment:
It adds 3kb (compressed) to main bundle, not blocking but would probably lazy load it as a whole https://vercel.com/dailydotdev/daily-webapp/HiiSd1hDdRx5BQGeah5XcDfvmHjY#L139

Reply:
We can do it as optimization :) (needs some testing I think with enrollment etc)